browser/js/orders/orders.js

```diff
@@ -1,17 +1,26 @@
-'user strict';
+'use strict';
 
 // Set up the state provider
 app.config(function ($stateProvider) {
 	$stateProvider.state('orders', {
 		url: '/orders',
 		templateUrl: 'js/orders/orders.html',
-		controller: 'OrdersCtrl'
+		controller: 'OrdersCtrl',
+		resolve: {
+			orderInfo: function (ordersFactory) {
+				return ordersFactory.getOrders("5541421a6c6c31d47876e032");
```

Comment:
Hardcoded to get orders for a single User right now, will update so that it derives the User from the sessionID.